lib/features/run_placed/presentation/ui/widgets/cancel_delivery_button.dart

```diff
@@ -14,6 +25,10 @@ class CancelDeliveryButton extends StatelessWidget {
         icon: FaIcon(FontAwesomeIcons.windowClose, color: Colors.white),
         onPressed: () {
           print("Cancel delivery button is pressed");
+          final returnVal = updateOrInsertRun(currRun.copyWith(status: RunStatus.CANCELLED));

```

Comment:
I'm wondering if we will have synchronization problem because of this syntax. If that happens we won't be able to avoid wrapping it with our own bloc.